src/utils/time/index.js

```diff
@@ -20,24 +20,24 @@ export const calculateTime = (time, formatForChatTab = false) => {
   }
 
   if (minutes < 60) {
-    return minutes === 0 ? '1m ago' : `${minutes}m ago`;
+    return minutes === 0 ? '1m' : `${minutes}m`;
   }
 
   if (hours < 24) {
-    return `${hours}h ago`;
+    return `${hours}h`;
   }
 
   if (days === 1) {
-    return formatForChatTab ? `${days}d ago` : 'Yesterday';
+    return formatForChatTab ? `${days}d` : '1d';
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from 'Yesterday' to '1d' might not be as intuitive for users. 'Yesterday' provides a clearer understanding that the event happened the previous day. Consider reverting this change unless there's a specific reason for it.
```diff
- return formatForChatTab ? `${days}d` : '1d';
+ return formatForChatTab ? `${days}d ago` : 'Yesterday';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->